annotation-client/src/test/java/eu/europeana/annotation/client/integration/webanno/transcribe/BaseTranscribingTest.java

```diff
@@ -10,6 +10,7 @@
 
 	public static final String TRANSCRIPTION_MINIMAL = "/transcription/minimal.json";
 	public static final String TRANSCRIPTION_WITH_RIGHTS = "/transcription/transcription-with-rights.json";
+	public static final String TRANSCRIPTION_WITHOUT_RIGHTS = "/transcription/transcription-without-rights.json";
```

Comment:
I don't see this file used, a unit test to check the the validation exception must be written

Reply:
I prepared this test, and see that server returns as expected ParamValidationException that extends HttpException. But Exception test fails for unclear reason with message: org.opentest4j.AssertionFailedError: Unexpected exception type thrown ==> expected: <org.apache.commons.httpclient.HttpException> but was: <org.opentest4j.AssertionFailedError> even when returned exception is correct.

Reply:
at the client side we don't have the Http Exception, it is enough to verify sta status code